### PR TITLE
Implement Selenium test orchestration

### PR DESCRIPTION
**Pull request by lucernae**
This PR integrate selenium driver stack for browser based testing in docker.

# Issue

for #83 and #82

### CODE REVIEW

File: deployment/docker/REQUIREMENTS-dev.txt [hunk 1]
```diff
@@ -2,11 +2,11 @@ flake8==3.7.7
 pycodestyle==2.5.0
 pyflakes==2.1.1
 mccabe==0.6.1
+django-test-without-migrations==0.6
```
Comment (lucernae) [inline deployment/docker/REQUIREMENTS-dev.txt:5]: I added this package to make test launch faster (skipping migrations).

File: django_project/core/settings/contrib.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from .base import *  # noqa
 
-# Extra installed apps - grapelli needs to be added before others
+# Extra installed apps that needs to be added before others
 INSTALLED_APPS = (
+    'test_without_migrations',
```
Comment (lucernae) [inline django_project/core/settings/contrib.py:5]: This package also needs to be loaded first.

File: deployment/docker/REQUIREMENTS.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 psycopg2-binary
-Django==2.1.5
+Django==2.1.7
```
Comment (lucernae) [inline deployment/docker/REQUIREMENTS.txt:2]: For Github security alert

File: django_project/ford3/tests/functional/test_homepage.py [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+
+    @unittest.skipUnless(
+        selenium_flag_ready(),
+        'Selenium test was not setup')
+    def test_homepage(self):
+        # test that driver perfectly configured
+        self.assertTrue(self.driver)
+
+        self.driver.get(self.live_server_url)
+        self.assertIn('FORD3', self.driver.title)
```
Comment (lucernae) [inline django_project/ford3/tests/functional/test_homepage.py:17]: I modified this page as a sample @pierrealixt

**Review by lucernae — COMMENTED** [review]
@pierrealixt please Review 📦

**Review by pierrealixt — APPROVED** [review]
LGTM!!

Good job!

**Pull request merged**